app/code/Dss/SalesOrderGrid/view/adminhtml/templates/sales/order/js.phtml

```diff
@@ -0,0 +1,19 @@
+<!-- /**
+ * Digit Software Solutions..
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the EULA
+ * that is bundled with this package in the file LICENSE.txt.
+ *
+ * @category   Dss
+ * @package    Dss_SalesOrderGrid
+ * @author     Extension Team
+ * @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+ */ -->
+<?php
+/** @var $block \Dss\SalesOrderGrid\Block\Adminhtml\Sales\Order\Js */
+?>
+<script>
```

Comment:
use as per coding standards for magneto script tag

Reply:
@harishmk1805 
changes uploaded please check